Source/Eto.WinForms/Drawing/GraphicsHandler.cs

```diff
@@ -69,12 +69,12 @@ public GraphicsHandler(sd.Graphics graphics, bool shouldDisposeGraphics = true)
 			this.shouldDisposeGraphics = shouldDisposeGraphics;
 		}
 
-		protected override void Dispose(bool disposing)
+		protected override bool DisposeControl
 		{
-			if (!shouldDisposeGraphics)
-				Control = null;
-
-			base.Dispose(disposing);
+			get
+			{
+				return shouldDisposeGraphics;
+			}
```

Comment:
This is just code cleanup. `DisposeControl` override has exactly this purpose, so why altering `Dispose(bool)` and doing it another way.